src/components/FormSpy.js

```diff
@@ -0,0 +1,12 @@
+import { FormSpy } from 'react-final-form'
+import propTypes from '@dhis2/prop-types'
```

Comment:
Since we're depending on it directly, it seems like a good idea to me to yarn add @dhis2/prop-types. Otherwise we're relying on ui-core bundling it.

Reply:
Out-of-scope for now and issue was created so we don't forget to work on this in the future https://github.com/dhis2/ui-forms/issues/141